sources/ClangSharp/Extensions/CXCompletionString.cs

```diff
@@ -14,12 +14,18 @@ public partial struct CXCompletionString
 
         public CXString GetAnnotation(uint index) => clang.getCompletionAnnotation(this, index);
 
-        public CXCompletionString GetChunkCompletionString(uint index) => clang.getCompletionChunkCompletionString(this, index);
+        public CXCompletionString GetChunkCompletionString(uint index) => (CXCompletionString)clang.getCompletionChunkCompletionString(this, index);
 
         public CXCompletionChunkKind GetChunkKind(uint index) => clang.getCompletionChunkKind(this, index);
 
         public CXString GetChunkText(uint index) => clang.getCompletionChunkText(this, index);
 
-        public CXString GetParent(ref CXCursorKind kind) => clang.getCompletionParent(this, ref kind);
+        public CXString GetParent(out CXCursorKind kind)
+        {
+            fixed (CXCursorKind* pKind = &kind)
```

Comment:
The downside (or upside, depending on how you look at it) to having unsafe bindings is you get to explicitly see all the "hidden" overhead that the marshaler was adding (such as implicitly pinning a value).